.travis.yml

```diff
@@ -5,12 +5,14 @@ env:
   global:
     - PIP_DOWNLOAD_CACHE=$HOME/.pip_cache
   matrix:
-    - TOX_ENV=py27-django15
-    - TOX_ENV=py27-django16
-    - TOX_ENV=py27-django17
-    - TOX_ENV=py34-django15
-    - TOX_ENV=py34-django16
-    - TOX_ENV=py34-django17
+    - TOX_ENV=py27-django111
+    - TOX_ENV=py27-django20
```

Comment:
Django 1.11 was the last version to support Python 2.7. If these are not breaking then I think we have an issue with our tox setup.